main.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 var METRIC_CHANNEL_POLL_INTERVAL = 2 * time.Second
-var TARGET_LIST_SCAN_WAIT_INTERVAL = 5 * time.Second
+var TARGET_LIST_SCAN_WAIT_INTERVAL = 4 * time.Second
 var PROBER_RESTART_INTERVAL_JITTER_RANGE = 2
-var METRIC_CHANNEL_SIZE = 1000
+var METRIC_CHANNEL_SIZE = 400
```

Comment:
Curious, what's the reasoning behind of reducing the channel size ?

Reply:
I was just playing around since we haven't picked the best size yet.

Reply:
👍